### ISSUE TITLE
rp2 as new or optional back-end?

### ISSUE THREAD

**Issue by ParaplegicRacehorse** [rotki/rotki#5729]
## Abstract

Add to or replace the back-end tracking and tax-solving with [rp2](https://github.com/eprbell/rp2) and [dali-rp2](https://github.com/eprbell/dali-rp2).

## Motivation

I want a nice front-end for tracking and tax-solving my crypto.  Currently, rotki supports less than 2% of my portfolio (in value; less than 0.5% in coins/tokens) and rp2 supports, roughly, 80% of my portfolio (coins/tokens.)  rotki is pretty and graphical.  rp2 is not.  rotki supports a tiny tiny number of blockchains, exchanges, wallets, national tax reporting structures, etc.  rp2, via dali-rp2, supports almost everything crypto.

In short, rotki gets, for free, a major boost to supported "stuff" and can focus on making the interface more usable.

**Comment by LefterisJP** [comment]
Thanks for the suggestion but this is not something we are considering doing.

rotki is expanding its supported protocols/chains/exchanges with each release and we support the most important chains, protocols and exhchanges. For those we don't yet support automatically you can add them manually: https://rotki.readthedocs.io/en/latest/usage_guide.html#adding-manual-balances
